example-plugins/patient_app_schedule/CANVAS_MANIFEST.json

```diff
@@ -0,0 +1,38 @@
+{
+  "sdk_version": "0.1.4",
+  "plugin_version": "0.0.1",
+  "name": "patient_app_schedule",
+  "description": "Edit the description in CANVAS_MANIFEST.json",
+  "components": {
+    "protocols": [
+      {
+        "class": "patient_app_schedule.protocols.my_protocol:Providers",
+        "description": "A protocol that does xyz...",
+        "data_access": {
+          "event": "",
```

Comment:
this question is bigger than your PR but I've been noticing that some plugins will list the event here (ex: PATIENT_PORTAL__APPOINTMENTS__FORM_PROVIDERS__POST_SEARCH) and others do not... so it seems like it's not required, is that correct? Should we be encouraging folks to fill it in anyway?